launch/move_base.launch

```diff
@@ -1,24 +1,28 @@
 <?xml version="1.0" ?>
 <launch>
-  <arg name="robotont_ns" default="robotont"/>
-    <!-- Launch navigation stack -->
+  <arg name="__ns" default=""/>
+  <!-- As eval may not contain double underscore expressions, a new argument is defined to by-pass this issue. -->
+  <arg name="eval_underscore_bypass" value="$(arg __ns)"/>
+  <!-- When a namespace is specified, it is also converted into prefix with a trailing underscore. -->
+  <arg if="$(eval eval_underscore_bypass == '')" name="prefix" default=''/> 
+  <arg unless="$(eval eval_underscore_bypass == '')" name="prefix" default="$(arg __ns)_"/>
+  <!-- Launch navigation stack -->
   <arg name="no_static_map" default="false"/>
+
   <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
-        <remap from="cmd_vel" to="$(arg robotont_ns)/cmd_vel"/>
-        <remap from="odom" to="$(arg robotont_ns)/odom"/>
         <rosparam file="$(find robotont_navigation)/config/move_base_params.yaml" command="load"/>
         <rosparam file="$(find robotont_navigation)/config/planner_local_params.yaml" command="load" />
         <rosparam file="$(find robotont_navigation)/config/planner_global_params.yaml" command="load" />
         <!-- observation sources located in costmap_common.yaml -->
-        <rosparam file="$(find robotont_navigation)/config/costmap_common.yaml" command="load" ns="global_costmap" />
-        <rosparam file="$(find robotont_navigation)/config/costmap_common.yaml" command="load" ns="local_costmap" />
+        <rosparam file="$(find robotont_navigation)/config/costmap_common.yaml" subst_value="true" command="load" ns="global_costmap" />
+        <rosparam file="$(find robotont_navigation)/config/costmap_common.yaml" subst_value="true" command="load" ns="local_costmap" />
```

Comment:
Just for the record, why subst_value param has to be enabled?

Reply:
i am guessing to pass prefix to costmap_common.yaml